mcs/class/corlib/Test/System.Reflection/PropertyInfoTest.cs

```diff
@@ -391,7 +391,12 @@ public class TestE {
 			var p = t.GetProperty ("Prop");
 			Assert.AreEqual ("test", p.GetConstantValue (), "#1");
 
-			File.Delete (Path.Combine (Path.GetTempPath (), an));
+			try {
+				// This throws an exception under MS.NET and Mono on Windows,
+				// open files cannot be deleted. That's fine.
+				File.Delete (Path.Combine (Path.GetTempPath (), an));
+			} catch (Exception) {
```

Comment:
The code deletes the file before saving the assembly: https://github.com/ntherning/mono/blob/c862a7b42378773333a4dae5f0dcd5626d6510f1/mcs/class/corlib/Test/System.Reflection/PropertyInfoTest.cs#L386

The file is also covered by `.gitignore`. So it should be ok.